bitnami/opensearch/Chart.yaml

```
@@ -30,4 +30,4 @@ maintainers:
 name: opensearch
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/opensearch
-version: 1.3.7
+version: 2.0.0
```

I would go with a patch instead:
```suggestion
version: 1.3.8
```

---

Sure, we can do that if you want. I just want to be clear that this might be a breaking change for some people, since the old config and new config will have different outcomes. While the old outcome was probably not the intended outcome, it's possible that people learned to expect these exact resource names and thus depend on that old incorrect behavior.

It's not so clear-cut whether bug fixes are breaking changes or not. If we had an app with an API and the API spec stayed the same but a bug was fixed, I think it's fair to call it a non-breaking change. But in our case, we have no API (or perhaps the whole chart can be considered the API spec), so it's hard to tell what's breaking or not.

---

But you are the maintainer, so if you want to call this a non-breaking change that's fine by me.